.travis.yml

```diff
@@ -53,8 +44,12 @@ env:
   - TARGET=":jsettlers.logic:test:aiDifficultiesIT"
   - TARGET=":jsettlers.logic:test:autoReplayIT"
   - TARGET=":jsettlers.logic:test:replayValidationIT"
-    
-  # Test release generation
+  
   - TARGET="release"
 
+  
 script: GRADLE_OPTS='-Xmx600m -Dorg.gradle.jvmargs="-Xmx1500m"' ./gradlew $TARGET
+
+
+after_success:
+- ./deploy.sh
```

Comment:
`after_success` is run in between of `script` and `after_script`. Have a look at the log output, it is called for every build you do ;-)